### ISSUE TITLE
In `top`, other user’s names becomes `nobody`

### ISSUE THREAD

**Issue by ayaka14732** [fsquillace/junest#275]
Steps to reproduce:

1. Open `top` command in junest
2. Check the `USER` column of the output. Other user’s names are shown as `nobody`

Is it possible to show the actual user names instead of `nobody`?

**Comment by fsquillace** [comment]
Yes, you are right. I am not sure why that happens. This happen with `ps` command too. Junest updates `/etc/passwd` file so I thought these programs use that file to get the username. We may need to update some other files.

If you have time to investigate this, please let me know, thanks.

**Comment by andryandrew** [comment]
I looked a little bit into it and [this](https://github.com/containers/bubblewrap/issues/574) could be the underlying issue (kernel namespace limitation)
